src/lib/Microsoft.Fx.Portability.MetadataReader/MemberMetadataInfo.cs

```diff
@@ -239,7 +239,7 @@ private IEnumerable<string> GetGenericDisplayNames(IList<string> displayNames)
         }
 
         /// <summary>
-        /// Replace the non-safe characters (<>.,) in a method name with safe ones ({}#@)
+        /// Replace the non-safe characters (.<>.,) in a method name with safe ones ({}#@)
```

Comment:
It looks like period is already in the list? Was this another application of the period rule?